src/Common/TurbulenceClosures/TurbulenceClosures.jl

```diff
@@ -402,6 +405,81 @@ function turbulence_tensors(
     return ν, D_t, τ
 end
 
+# ### [Viscous Sponge](@id viscous-sponge)
+# `ViscousSponge` requires a user to specify a constant viscosity (kinematic), 
+# a sponge start height, the domain height, a sponge strength, and a sponge
+# exponent.
+# It works like `ConstantViscosityWithDivergence` but without divergence and is
+# typically used at the top of the domain to absorb waves. A smooth onset is
+# ensured through a weight function that increases weight height from the sponge
+# onset height.
+# ```
+"""
+    ViscousSponge <: TurbulenceClosureModel
+
+Sponge layer with constant dynamic viscosity (`ρν`).
+
+# Fields
+
+$(DocStringExtensions.FIELDS)
+"""
+struct ViscousSponge{FT} <: TurbulenceClosureModel
+    "Dynamic Viscosity [kg/m/s]"
```

Comment:
Do you intend to use this in addition to one of these turb models ? If this is the case then setting this up as a source term might be more convenient (\nabla u is available through the state_gradient_flux variables in the source function)

Reply:
Ok, so we move this over to source? @akshaysridhar, do we need anything else?